python/spacewalk/satellite_tools/reposync.py

```diff
@@ -243,6 +243,20 @@ def get_addons(self):
 
         return addons_dirs
 
+    def remove_variant(self, variant):
+        try:
+            variants = self.parser.get("tree", "variants").split(",")
+            if "AppStream" in variants:
+                variants.remove("AppStream")
```

Comment:
What will happen with the treeinfo of AppStream? AFAIK we want to remove it only from BaseOS

Reply:
right! I forgot that one... I'll handle that

Reply:
Looks like I had a hard time focusing on code yesterday... it's not the only broken thing. I'll propose a better solution.